phases/phase-1/README.md

```diff
@@ -3,3 +3,9 @@ The first phase of the Primer is based on the `morpheus-1000` testnet. During th
 
 - [Setup](setup/README.md)
 - [Challenges](challenges/README.md)
+
+:::warning Phase 1 ending  
+Phase 1 will close on Friday 31th January 2020 at 23:59:59 UTC.  
+
+Submissions after that date will **not** receive any reward.  
```

Comment:
Submissions after that date will **not** receive any reward.

->

Submissions after that date may **not** receive any reward.

(Reason: using the word "may" has given us sufficient protection while easing the hard feelings of the potential users.)